assets/templates.go

```diff
@@ -110,7 +110,7 @@ func templatesDatasetFilterAgeTmpl() (*asset, error) {
 		return nil, err
 	}
 
-	info := bindataFileInfo{name: "templates/dataset-filter/age.tmpl", size: 8419, mode: os.FileMode(420), modTime: time.Unix(1541062665, 0)}
+	info := bindataFileInfo{name: "templates/dataset-filter/age.tmpl", size: 8419, mode: os.FileMode(420), modTime: time.Unix(1543230477, 0)}
```

Comment:
Should this file be committed? Am I right in thinking it's generated?

Reply:
Yeah it needs to be pushed :)